app/components/VaultControls/index.js

```diff
@@ -1086,6 +1124,24 @@ export default function VaultControls(props) {
                   />
                 </Box>
                 <ButtonGroup width={1}>
+                  {isScreenMd ? (
+                    <Box center mr={15} ml={2} style={{ width: '15px' }}>
+                      <svg
+                        width="24"
+                        height="24"
+                        viewBox="0 0 24 24"
+                        fill="none"
+                        xmlns="http://www.w3.org/2000/svg"
+                      >
+                        <path
+                          fillRule="evenodd"
+                          clipRule="evenodd"
+                          d="M19.7593 12.6508C20.0802 12.2763 20.0802 11.7237 19.7593 11.3492L13.7593 4.34923C13.3998 3.9299 12.7685 3.88134 12.3492 4.24076C11.9299 4.60018 11.8813 5.23148 12.2407 5.65081L17.6829 12L12.2407 18.3492C11.8813 18.7686 11.9299 19.3999 12.3492 19.7593C12.7685 20.1187 13.3998 20.0701 13.7593 19.6508L19.7593 12.6508ZM6.75926 19.6508L12.7593 12.6508C13.0802 12.2763 13.0802 11.7237 12.7593 11.3492L6.75926 4.34923C6.39983 3.9299 5.76853 3.88134 5.34921 4.24076C4.92988 4.60018 4.88132 5.23148 5.24074 5.65081L10.6829 12L5.24074 18.3492C4.88132 18.7686 4.92988 19.3999 5.34921 19.7593C5.76853 20.1187 6.39983 20.0701 6.75926 19.6508Z"
+                          fill="white"
+                        />
+                      </svg>
+                    </Box>
+                  ) : null}
```

Comment:
Could we abstract this into a component (e.g. DoubleArrowRightIcon) so we don't add many extra lines to this super large file?

Reply:
ok, pushed